itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -186,6 +186,8 @@ static void internalBeforeClassSetup(Map<String, String> additionalProperties, b
     });
 
     MetaStoreTestUtils.startMetaStoreWithRetry(hconf);
+    // re set the WAREHOUSE property to the test dir, as the previous command added a random port to it
+    hconf.set(MetastoreConf.ConfVars.WAREHOUSE.getVarname(), System.getProperty("test.warehouse.dir", "/tmp"));
```

Comment:
Yes it wouldn't in theory, still the class already references this property 4 times before this change, twice specifying /tmp as fallback value, twice not. I thought that we should be consistent, and it can't hurt to have it there so I put it here, and also fixed the missing one in line 3273 - but missed to do so in the 4th occasion at line 3293. So we can have it at all occasions,, or remove it from all, just be consistent. What do you suggest?

Reply:
Consistency is good idd... We can include `tmp` it in all occurrences.

Reply:
Agree, fixed.